src/platform/site-wide/user-nav/containers/Main.jsx

```diff
@@ -39,11 +39,109 @@ import {
 } from 'platform/site-wide/user-nav/actions';
 import { updateLoggedInStatus } from 'platform/user/authentication/actions';
 import { ACCOUNT_TRANSITION_DISMISSED } from 'platform/user/authentication/constants';
+import { VaModal } from '@department-of-veterans-affairs/component-library/dist/react-bindings';
+import * as authUtilities from 'platform/user/authentication/utilities';
 import SearchHelpSignIn from '../components/SearchHelpSignIn';
 import AutoSSO from './AutoSSO';
 import { selectUserGreeting } from '../selectors';
+import recordEvent from '../../../monitoring/record-event';
+
+// remove this when organic experiment finishes
+const CreateLoginGovAccountModal = ({ visible = false, onClose }) => {
+  let logingovSingUpLink;
+  // uses logic from CreateAccountLink component
+  async function generateURL() {
+    const url = await authUtilities.signupOrVerify({
+      policy: 'logingov',
+      isLink: true,
+      allowVerification: false,
+      useOAuth: true,
```

Comment:
There's no guarantee that the user will be signed with OAuth you'll need to conditonalize this, most likely through a prop (see line 244, on how we do this with the current SignInModal component)